src/components/calcite-button/calcite-button.stories.js

```diff
@@ -46,22 +47,46 @@ storiesOf("Button", module)
     "With icon",
     () => `
     <calcite-button
-      icon="${text("icon", images24)}"
+      appearance="${select(
+        "appearance",
+        {
```

Comment:
when the property is the same as the value the options can be an array like `["solid", "clear", "inline", "outline", "transparent"]` IIRC

Reply:
Oh, rad. Will update ⚡️